tests/contrib/hooks/test_gcp_cloud_build_hook.py

```diff
@@ -46,7 +47,7 @@
 
 
 class TestCloudBuildHookWithPassedProjectId(unittest.TestCase):
-    hook = None
+    hook = None  # type: Optional[CloudBuildHook]
```

Comment:
Similar here - although this is `None` here it is always set to a CloudBuildHook for every test case (cos of setUp). I forget how mypy behaves with none-but-initalized-later...?

Reply:
replied in the other comment :)